JASP-Engine/JASP/R/correlation.R

```diff
@@ -698,6 +691,13 @@ Correlation <- function(dataset=NULL, options, perform="run", callback=function(
 					 } else {
 
 						if (perform == "run") {
+						  errors <- .hasErrors(dataset, perform = "run", message = 'short', type = c('variance', 'infinity'),
```

Comment:
I think it's better to generally stick to `perform = perform` rather than a hard-coded value. If the if-statement above it is ever changed this might lead to unexpected behavior (same for other files).